janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -943,7 +943,7 @@ private void setCFOptions(HColumnDescriptor cdesc, int ttlInSeconds) {
                             // @see JanusGraphManagement#setTTL(JanusGraphSchemaType, Duration)
                             // Cast Put to Mutation for backward compatibility with HBase 0.98.x
                             // HBase supports cell-level TTL for versions 0.98.6 and above.
-                            ((Mutation) putColumnWithTtl).setTTL(ttl * 1000);
+                            ((Mutation) putColumnWithTtl).setTTL((long)ttl * 1000);
```

Comment:
Not necessary but I am suggesting to use `TimeUnit.SECONDS.toMillis(ttl)` which is a cleaner way to convert between time units.